### ISSUE TITLE
BugLife barTintColor doesn't seem to be working

### ISSUE THREAD

**Issue by rickvdl** [Buglife/Buglife-iOS#22]
Hi,

I'm trying to give the bug reporter's navigation bar a barTintColor, but I can't seem to get it working.
I am using the following code for updating the appearance:
```
let appearance = Buglife.shared().appearance
appearance.tintColor = UIColor.red
appearance.barTintColor = Colors.black
appearance.titleTextAttributes = [NSForegroundColorAttributeName : UIColor.red]
appearance.statusBarStyle = .lightContent
Buglife.shared().invocationOptions = [.screenshot]
Buglife.shared().start(withAPIKey: Constants.kBugLifeAPIKey)        
```

Using the following code I expect the navigationbar to be black with red textlabels. But this is the result:

![simulator screen shot 19 mar 2017 19 40 15](https://cloud.githubusercontent.com/assets/6663039/24083686/3cd4ecb2-0cdc-11e7-8ab6-dd65b830740f.png)

Any idea on what I'm doing wrong? 
Thanks in advance

**Comment by schukin** [comment]
Are you using UINavigationBar's appearance proxy anywhere in your application code to change its background image? (i.e. `UINavigationBar.appearance().setImage(for:)`)

**Comment by rickvdl** [comment]
I do, I am setting the background to an empty UIImage to hide the bottom divider.

**Comment by schukin** [comment]
Gotcha. In that case, this appears to be a bug in Buglife SDK. (Oh, the irony...)

We'll release an update tomorrow with a fix for this. :) However, if you require a fix immediately, you can work around this issue by setting the background image to nil specifically for Buglife navigation bars:

```swift
let buglifeNavBarType = NSClassFromString("LIFENavigationBar") as! UINavigationBar.Type
buglifeNavBarType.appearance().setBackgroundImage(nil, for: .default)
buglifeNavBarType.appearance().setBackgroundImage(nil, for: .compact)
```

**Comment by rickvdl** [comment]
Thanks for the quick response! Tomorrow should be fine, thanks for the snippet though!

Cheers

**Comment by schukin** [comment]
This issue has been resolved in Buglife v1.5.3. Cheers!
